packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -67,7 +67,8 @@ export const getRetryDelayFromError = (error: any): number | undefined => error?
 
 function getNackReconnectInfo(nackContent: INackContent) {
     const reason = `Nack: ${nackContent.message}`;
-    const canRetry = ![403, 429].includes(nackContent.code);
+    const canRetry = (nackContent.code === 429 && nackContent.retryAfter !== undefined)
```

Comment:
Should we also check whether retryAfter is positive or not?

Reply:
My preference would be not to check retryAfter in this logic. If it's not there, we will retry right away.
Note that in other places (see default case in createOdspNetworkError()) we mark non-retryable errors, and allow anything unknown to be retried, I feel it would be useful to do same here, i.e. simply remote 429 from that list - that makes your change way safer (as I do not understand all the other codes that might be used by either service)